Makefile

```diff
@@ -6,20 +6,23 @@ IMAGE_TAG := "latest"
 
 OUT := $(shell pwd)/_out
 
-KUBEBUILDER_VERSION=2.3.2
+KUBE_VERSION=1.21.2
 
 $(shell mkdir -p "$(OUT)")
+export TEST_ASSET_ETCD=_test/kubebuilder/bin/etcd
+export TEST_ASSET_KUBE_APISERVER=_test/kubebuilder/bin/kube-apiserver
+export TEST_ASSET_KUBECTL=_test/kubebuilder/bin/kubectl
```

Comment:
No need to change it here, but FYI there's 

> KUBEBUILDER_ASSETS | path to directory | Point integration tests to a directory containing all binaries (api-server, etcd and kubectl).

 * https://book.kubebuilder.io/reference/envtest.html#environment-variables